src/controllers/application-type.controller.js

```diff
@@ -0,0 +1,91 @@
+const HttpStatus = require('http-status-codes')
+const util = require('../util/util.helpers')
+
+module.exports = appTypeService => {
+  function getAllApplicationTypes (req, res, next) {
+    const setResponse = appTypes => {
+      res.status(HttpStatus.OK)
+        .pagedJson(appTypes)
+    }
+
+    const { page, pageSize } = req.query
+
+    return appTypeService.getAll(page, pageSize)
+      .then(setResponse)
+      .catch(next)
+  }
+
+  function getByApplicationType (req, res, next) {
+    const setResponse = appType => {
+      if (!appType) {
+        let message = 'Application Type not found.'
+        return util.generateError(message, HttpStatus.NOT_FOUND)
+      }
+
+      res.status(HttpStatus.OK)
+        .json(appType)
+    }
+
+    const { id } = req.params
+
+    return appTypeService.getById(id)
+      .then(setResponse)
+      .catch(next)
+  }
+
+  function setApplicationType (req, res, next) {
+    const setResponse = appType => {
+      res.status(HttpStatus.OK)
+        .json(appType)
+    }
+
+    const appType = req.body
+
+    return appTypeService.create(appType)
+      .then(setResponse)
+      .catch(next)
+  }
+
+  function updateApplicationType (req, res, next) {
+    const setResponse = appType => {
+      res.status(HttpStatus.OK).json()
+    }
+
+    const getAppType = appType => {
+      if (!appType) {
+        let message = 'Application Type not found.'
+        return util.generateError(message, HttpStatus.NOT_FOUND)
+      }
+
+      return appTypeService.update(appType, values)
```

Comment:
Place this constant inside `getAppType()`. It's the only place you're using it and we should keep our functions pure when possible.